### PR TITLE
personal detail page

### PR DESCRIPTION
**Pull request by Shaktiexcellence**
Personal details

### CODE REVIEW

File: lib/screens/navigate/dropdown_inventory.dart [hunk 1]
```diff
@@ -8,10 +8,11 @@ class DropDown extends StatefulWidget {
 
 class _DropDownState extends State<DropDown> {
   List<ListItem> _dropdownItems = [
-    ListItem(1, "Select Your Audit Report "),
-    ListItem(2, "Second Item"),
-    ListItem(3, "Third Item"),
-    ListItem(4, "Fourth Item")
+    ListItem(1, "Single "),
+    ListItem(2, "Married"),
```
Comment (manishiitg) [inline lib/screens/navigate/dropdown_inventory.dart:12]: all hard coded should be moved a single configuration file. it should not be in code at all

File: lib/screens/profile/edit_profile.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          pageName: "Edit profile",
+        ),
+      ),
+      drawer: Navigation(),
+      body: SafeArea(      
+          child: SingleChildScrollView(
+        child: Column(
+          children: <Widget>[
+            EditNotice(),
+            IdCard(name: " Shubham Mishra", designation:"Jr.Mobile App Developer", empId:"523",joinDate:"2020-08-10",workEmail: "shubham@excellence.in",gender: "Male",dob: "1997-09-06",contact: "9999999988",),
```
Comment (manishiitg) [inline lib/screens/profile/edit_profile.dart:23]: ?

Comment (Shaktiexcellence) [reply lib/screens/profile/edit_profile.dart:23]: means ?

File: lib/screens/navigate/dropdown_inventory.dart (hunk 1 above)
Comment (Shaktiexcellence) [reply lib/screens/navigate/dropdown_inventory.dart:12]: I designed static Dropdown , @deepaketech make me understand How to do.???

**Pull request merged**